modules/ROOT/pages/setup.adoc

```diff
@@ -3,25 +3,30 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Set up Partner Manager to be used by different people in your organization: message flow developers, operations managers, and IT managers.
+Set up Anypoint Partner Manager to be used by different people in your organization: message flow developers, operations managers, and IT managers.
 To configure the system, you should be familiar with these concepts before setting up Partner Manager:
 
-* EDI message formats and schemas.
-* Dataweave transformations.
-* Digital Certificates such as X.509 certificates in PKCS12 file format for the AS2 transport protocol.
-* Configuration of protcols such as AS2, SFTP, HTTPS.
-* Understanding ofthe concept of organizations and environments in the Anypoint platform in context to Access Management and Anypoint Runtime Manager.
+* EDI message formats and schemas
+
+* Dataweave transformations
+
+* Digital Certificates such as X.509 certificates in PKCS12 file format for the AS2 transport protocol
+
+* Configuration of protcols such as AS2, SFTP, and HTTPS
+
+* The concept of organizations and environments in the Anypoint platform in context to Access Management and Mule Runtime Manager
 
 == Minimum Requirements
 
 Verify that you have the following software and other requirements:
 
-* Mule runtime engine 4.1.1 or higher. Make sure you have the needed Mule runtime Core/vCore capacity to handle your B2B transaction workload needs.
+* Mule runtime engine 4.1.1 or higher. Make sure you have the needed Mule runtime Core/vCore capacity to handle your B2B transaction workload needs
+
 If you are unsure about the capacity you need to handle your B2B transactions, please contact your MuleSoft account representative for assistance.
 
 * A storage solution to store the B2B transmission payloads your send or receive. Storage options can include a Database, Amazon S3, Azure Blob Storage, and so on.
 
-* Anypoint Studio 7.1 or higher to create DataWeave maps.
+* Anypoint Studio 7.1 or higher to create DataWeave maps
```

Comment:
```suggestion
* Anypoint Studio 7.1 or later to create DataWeave maps
```

Already edited this above. I am confused by repeated text. Please do not include it in PRs.